.github/ISSUE_TEMPLATE/new_release_repository.md

```diff
@@ -7,8 +7,18 @@ title: 'Add new release repositories'
 * Name of the release team: 
 
 * Release repositories to add:
+  <!--
+  Before release repositories are created, repositories and packages should be submitted to the official rosdistro repository https://github.com/ros/rosdistro for naming review.
+  The ros/rosdistro source entry link must be included for each repository.
+  Guidelines for package naming are describe in REP-144: https://www.ros.org/reps/rep-0144.html
+
+  If there are any existing release repositories which should have their contents imported into the ros2-gbp organization, list and link to them here.
+  Leave the checkbox _unchecked_. The ros2-gbp administrator will check the box when they have completed the repository import.
+  -->
   * [FIRST_REPOSITORY](SOURCE URL)
+    * [ros/rosdistro source entry](LINK TO SOURCE FIELD IN ROS/ROSDISTRO)
```

Comment:
I didn't understand what "LINK TO SOURCE FIELD" points to.

Should a URL to a PR that is already opened in rosdistro be sufficient? With that, it'll be ros2-gbp maintainer's job to find the info that this line is currently asking for unfortunately, but as a total that might reduce confusion and make the process smoother.

Reply:
I agree this step could step was confusing both from the instructions on docs.ros.org here:
- https://docs.ros.org/en/rolling/How-To-Guides/Releasing/Release-Team-Repository.html#create-a-new-release-repository
and from this template.

I think it should suffice to have a link to an open PR otherwise the process could just take too long, waiting for maintainers of both ros/rosdistro and ros2-gbp/ros2-gbp-github-org

- https://github.com/ros2-gbp/ros2-gbp-github-org/issues/283
- https://github.com/ros/rosdistro/pull/37736

Reply:
> I think it should suffice to have a link to an open PR otherwise the process could just take too long, waiting for maintainers of both ros/rosdistro and ros2-gbp/ros2-gbp-github-org

Thanks for the feedback. Just opening the ros/rosdistro PR is not sufficient to indicate that a newly added repository has actually passed any kind of review, so this change to the process would not be sufficient and we would end up with the same cases which caused us to introduce the source repository requirement in the first place: packages being submitted which do not meet rosdistro naming requirements, needing a name change, and having to do that name change _after_ having already created a ros2-gbp release repository.

The current process is clunky and far from optimal. I have been working on a new bloom subcommand which automates submitted PRs for source repository indexing in the same fashion that it can currently automatically create release PRs and one potential feature of this project in the future is the automatic creation and deployment of ros2-gbp release repositories as soon as the source repository is added to ros/rosdistro. I appreciate the feedback immensely because it helps me better understand which areas of the process are the most in-need of optimizations.